### ISSUE TITLE
BugFix: Excluded box is out of window if exclusion label is longer

### ISSUE THREAD

**Issue by vmrothe** [bruigtp/flowchart#13]
Hi, 
it's me again, sorry :D 
I just noticed, that exclusion boxes are cut off/ only partially displayed if the label of the exluded box is longer than one word. Please find an example here:

```
iris %>% 
  as_fc() %>% 
  fc_filter(Sepal.Length > 4.8, label="Sepal length > 4.8", show_exc = TRUE) %>% 
  fc_split(Species) %>% 
  fc_filter(Petal.Length < 6.5, label="Petal.Length < 6.5", show_exc = TRUE, label_exc = "Petal length >= 6.5" ) %>% 
  fc_draw()
```

Could you please adapt the spacing around the flowchart depending on the width of the exclusion boxes? Or am I missing an option to do that? 

Thanks already in advance!

**Comment by pasahe** [comment]
Dear @vmrothe,

Thanks for the post. The left and right margins I set for the exclusion boxes were too small, so as you noted, in some cases the exclusion boxes were cut off. I have changed the margin to allow more space in the new development version posted on github. Hopefully this solves your problem. In the future, we will explore the possibility of adding a parameter to control the distance for the excluded box for more flexibility and control over the drawn flowchart.

Thank you very much!
